packages/forma-36-react-components/src/components/Button/Button.stories.tsx

```diff
@@ -1,47 +1,79 @@
-import { storiesOf } from '@storybook/react';
-import { action } from '@storybook/addon-actions';
-import { text, boolean, select } from '@storybook/addon-knobs';
+
+import Button, { ButtonProps, buttonTypes } from './Button';
 import { iconName } from '../Icon/constants';
 
-import Button from './Button';
 import notes from './README.mdx';
 
-storiesOf('Components/Button', module)
-  .addParameters({
-    propTypes: Button['__docgenInfo'],
-    component: Button,
-  })
-  .add(
-    'Button',
-    () => (
-      <Button
-        className={text('className', '')}
-        icon={select('icon', [undefined, ...Object.keys(iconName)], undefined)}
-        buttonType={select(
-          'buttonType',
-          {
-            muted: 'muted',
-            primary: 'primary',
-            positive: 'positive',
-            negative: 'negative',
-            naked: 'naked',
-            warning: 'warning',
-          },
-          'muted',
-        )}
-        size={select('size', [undefined, 'small', 'large'], undefined)}
-        loading={boolean('loading', false)}
-        indicateDropdown={boolean('indicateDropdown', false)}
-        disabled={boolean('disabled', false)}
-        isFullWidth={boolean('isFullWidth', false)}
-        onClick={action('onClick')}
-        onBlur={action('onBlur')}
-        href={text('href', '')}
-        isActive={boolean('isActive', false)}
-      >
-        {text('Text', 'Embed entry')}
-      </Button>
-    ),
-    { notes },
-  );
+export default {
+  title: 'Components/Button',
+  component: Button,
+  parameters: {
+    propTypes: [Button['__docgenInfo']],
+    notes,
+  },
+  argTypes: {
+    icon: { control: { type: 'select', options: iconName } },
+    buttonType: { control: { type: 'select', options: buttonTypes } },
+    size: {
+      control: { type: 'select', options: [undefined, 'small', 'large'] },
+    },
+    className: { control: { disable: true } },
+    testId: { control: { disable: true } },
+    onClick: { control: { disable: true } },
+    onBlur: { control: { disable: true } },
+    loading: { control: { disable: true } },
+  },
+};
+
+export const basic = (args: ButtonProps) => (
+  <Button buttonType={args.buttonType} size={args.size}>
+    {args['Button Text']}
+  </Button>
+);
+
+export const typesOverview = (args: ButtonProps) => (
+  <>
+    <Button buttonType="primary">primary</Button>
+    <Button buttonType="positive">posiitve</Button>
+
+    <Button buttonType="warning">warning</Button>
+    <Button buttonType="negative">negative</Button>
+    <Button buttonType="muted">muted</Button>
+    <Button buttonType="naked">naked</Button>
+  </>
+);
+
+export const sizesOverview = (args: ButtonProps) => (
+  <>
+    <Button buttonType="primary" size="small">
+      small
+    </Button>
+    <Button buttonType="primary">default size</Button>
+    <Button buttonType="primary" size="large">
+      large
+    </Button>
+  </>
+);
+
+export const statesOverview = (args: ButtonProps) => (
```

Comment:
Or maybe we would prefer to have just one story with the overview of all the variants and states?

Reply:
Don't many stories work best with Chromatic?

Reply:
Maybe let's try to merge this one and I will create PR with some changes so we would be able to see it in Chromatic? WDYT?

Reply:
You can already see this in Chromatic here: https://www.chromatic.com/pullrequest?appId=5fd1dda724cc620021ace8c5&number=721&view=changes (click details on UI Review GitHub check).

I think having a single story would be nicer for Storybook users. It shouldn't matter too much as Chromatic will still pick up everything, I just guess the visual diffs might be a bit noisier.

Reply:
Sorry, didn't see your comment. Ok, sure, I will merge them into one story.

Reply:
I think I'm misunderstanding something, I only see a single button in the Basic story, is that right? How can Chromatic detect changes in the other types then?